pkg/eks/components/__tests__/Config.test.ts

```diff
@@ -107,11 +106,11 @@ describe('eKS K8s configuration', () => {
     expect(wrapper.exists()).toBe(true);
     expect(spy).toHaveBeenCalledTimes(0);
     await setCredential(wrapper);
-    expect(spy).toHaveBeenCalledTimes(4);
+    expect(spy).toHaveBeenCalledTimes(8);
```

Comment:
these are doubled because for each paginated aws api call we call `dispatch` twice: once to get the relevant aws client and once more to use the client in the `depaginate` action